recipe/meta.yaml

```diff
@@ -31,12 +31,13 @@ test:
     - xmlwf -h
 
 about:
-  home: http://expat.sourceforge.net/
+  home: https://expat.sourceforge.net/
   license: MIT
   license_family: MIT
   license_file: COPYING
```

Comment:
This should be corrected to `expat/COPYING`.

Reference: https://github.com/libexpat/libexpat/blob/R_2_4_4/expat/COPYING